Dockerfile.buster

```diff
@@ -54,8 +54,13 @@ RUN \
   && echo "deb-src https://deb.nodesource.com/${NODE_VERSION} buster main" | tee -a /etc/apt/sources.list.d/nodesource.list \
   && apt-get update \
   && apt-get install -y --no-install-recommends nodejs \
+  && apt-get install -y mariadb-server \
```

Comment:
do we need mariadb-server here or just `libmariadb3`?

we'll only want to install what is needed for building the Python package.

Reply:
Yeah, you're right, just tested and that was unnecessary to run the tests. Thanks for the catch.